src/sentry/static/sentry/app/components/charts/worldMapChart.tsx

```diff
@@ -124,7 +137,7 @@ class WorldMapChart extends React.Component<Props, State> {
     return (
       <BaseChart
         options={{
-          backgroundColor: theme.backgroundSecondary,
+          backgroundColor: theme.white,
```

Comment:
Won't this be problematic with dark mode? I think you want `theme.background` here.

Reply:
I haven't audited Dashboards v2 in dark mode yet. But I've applied your suggestion.